docs/operate/get-started/basics.md

```diff
@@ -0,0 +1,98 @@
+
+`viam-server` connects your machine to the Viam app and to SDK clients.
+
+See [Viam architecture](/operate/reference/architecture/) for details on how Viam works.
+
+## What is a machine?
+
+A _machine_ is a computer (often a single-board computer like a Raspberry Pi or Jetson) or microcontroller and all the hardware attached to it, as well as the software running on it.
+You can think of one machine as representing one device, such as an Intel RealSense camera connected to a Raspberry Pi.
+Each machine runs an instance of `viam-server`.
+
+When you create a new machine in the Viam app, Viam generates a unique set of credentials for that machine that connect the physical machine to its instance in the Viam app.
+
+## What platforms does Viam run on?
+
+`viam-server` can run on any computer that runs one of the following operating systems:
+
+- Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
+- macOS
+- Windows
+
+Viam also supports 32-bit microcontrollers such as the ESP32 series.
+
+Examples of computing devices that Viam supports:
+
+{{< board-carousel >}}
+
+## What hardware does Viam support?
+
+In short, any type of hardware can be integrated with Viam.
+
+Viam supports a wide variety of sensors, cameras, and other physical hardware, with APIs for each of the following types of hardware:
+
+{{< cards >}}
+{{% relatedcard link="/dev/reference/apis/components/arm/" %}}
+{{% relatedcard link="/dev/reference/apis/components/base/" %}}
+{{% relatedcard link="/dev/reference/apis/components/board/" %}}
+{{% relatedcard link="/dev/reference/apis/components/button/" %}}
+{{% relatedcard link="/dev/reference/apis/components/camera/" %}}
+{{% relatedcard link="/dev/reference/apis/components/encoder/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gantry/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gripper/" %}}
+{{% relatedcard link="/dev/reference/apis/components/input-controller/" %}}
+{{% relatedcard link="/dev/reference/apis/components/motor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/movement-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/power-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/servo/" %}}
+{{% relatedcard link="/dev/reference/apis/components/switch/" %}}
+{{% relatedcard link="/dev/reference/apis/components/generic/" %}}
+{{< /cards >}}
+
+These standardized APIs are implemented by {{< glossary_tooltip term_id="module" text="modules" >}} that provide drivers for specific models of hardware.
+
+Any hardware that is not already supported by a Viam module can be added into Viam's system of modular resources by [creating a new module](/operate/get-started/other-hardware/) that provides a driver for the hardware.
+
+## How do I get started?
+
+1. [Set up your machine and connect it to the Viam app](/operate/get-started/setup/).
```

Comment:
One final thought here: if we phrase this as "Set up a computer or SBC or set up an ESP32 and connect it to the Viam app", we could put the two respective links on the text that matches the left nav titles

Reply:
```suggestion
1. [Set up a computer or SBC](/operate/get-started/setup/) or [set up an ESP32](/operate/get-started/setup-micro/) and connect it to the Viam app.
```